packages/calcite-components/src/components/block-section/block-section.scss

```diff
@@ -1,7 +1,20 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-block-section-background-color: Specifies the component's background color.
+ * @prop --calcite-block-section-border-color: Specifies the border color of the component. Applicable on bottom border when open.
```

Comment:
I'm not sure we need the "Applicable on..." but will defer to @DitwanP / @geospatialem for consistency.

Reply:
```suggestion
 * @prop --calcite-block-section-border-color: Specifies the component's border color. When `open`, applies to the component's bottom border.
```